snapshots/collections-schema.yml

```diff
@@ -76,6 +76,58 @@ collections:
       versioning: false
     schema:
       name: gp_credits
+  - collection: gp_credits_additions
```

Comment:
I guess the columns `date_created` in `gp_credits` and `date_updated` in `gp_credits_additions` are useless now and can be removed as well.